cpp/src/barretenberg/plonk/composer/composer_lib.hpp

```diff
@@ -9,6 +9,57 @@ struct SelectorProperties {
     bool requires_lagrange_base_polynomial = false;
 };
 
+/**
```

Comment:
These two functions were formerly in the shared composer lib but they are used only for plonk so they've been moved to the plonk specific composer lib